pkg/apis/objectbucket.io/v1alpha1/objectbucket_types.go

```diff
@@ -109,6 +110,12 @@ type ObjectBucketStatus struct {
 	Conditions corev1.ConditionStatus  `json:"conditions"`
 }
 
+const ObjectBucketKind = "ObjectBucket"
```

Comment:
nit, but this is kind of awkwardly placed between ObjectBucket components.  can you move it to the top or bottom of the file?